.github/workflows/tests.yml

```diff
@@ -4,7 +4,7 @@ on: [ push ]
 
 jobs:
   lint:
-    if: "!contains(github.event.head_commit.message, 'skip ci')"
+    if: "contains(github.event.head_commit.message, 'Trigger CI')"
```

Comment:
can this be made case insensitive?

Reply:
The function contains is case insensitive by default.